scripts/imgtool/main.py

```diff
@@ -105,6 +107,27 @@ def validate_header_size(ctx, param, value):
             "Minimum value for -H/--header-size is {}".format(min_hdr_size))
     return value
 
+def get_dependencies(ctx, param, value):
+    if value is None:
+        return None
+    else:
+        versions = []
+        images = re.findall("\((\d+)", value)
+        raw_versions = re.findall(",\s*([0-9.+]+)\)", value)
+        if len(images) != len(raw_versions):
+            raise click.BadParameter(
+                "There's a mismatch between the number of \
+                dependency images and versions in: {}".format(value))
+        for raw_version in raw_versions:
+            try:
+                versions.append(decode_version(raw_version))
+            except ValueError as e:
+                raise click.BadParameter("{}".format(e))
+        if len(images) != 0:
+            dependencies = dict()
+            dependencies["images"] = images
+            dependencies["versions"] = versions
```

Comment:
Instead of using `"images"` and `"versions"` in all the dict accesses please add a const or helper for this...

Reply:
```
IMAGES = "images"
VERSIONS = "versions"
```
or
```
DEP_IMAGES = "images"
DEP_VERSIONS = "versions"
```
maybe? Or any other suggestion?

Reply:
I would probaby use something like:

```
DEP_IMAGES_KEY = "images"
DEP_VERSIONS_KEY = "versions"
```
but all look good, as longs as it's not just text copy/pasted around...

Reply:
I like the "_KEY" suffix, I'm going to update the script with it.